earthmover/earthmover.py

```diff
@@ -22,6 +23,7 @@
 
 from typing import List, Optional
 
+COMPILED_YAML_FILE = "./earthmover_compiled.yaml"
```

Comment:
I love globalizing these variables. Let's make it an Earthmover class attribute such that we can make it mutable via CLI one day.